src/pages/blog/kata-community-ptg-updates-april-2025-part-2.md

```diff
@@ -0,0 +1,76 @@
+
+The current proposal is to bump the Go version every 6 months to stay on a stable release at all times, while for things like MSRV only bump the version when needed, which would follow the process that the Confidential Containers (CoCo) project is applying. As bumping these versions can affect downstream users, the community is looking for a middle ground where they can move forward in a safe and stable way.
+
+It also came up in the past to use Dependabot to bump versions, but the community had issues with it in the past. While increased automation is desired in the processes, contributors still need to sort out some details on how to use and configure Dependabot for that.
+
+# Clean Interfaces Between Kata and Confidential Containers
+
+[Confidential Containers](https://confidentialcontainers.org) is a sandbox project in the CNCF ecosystem, which grew out of a working group under the Kata Containers project. CoCo is relying on Kata Containers, and therefore it is tightly integrated with Kata.
+
+This conversation focused largely on the use case of running Kata in confidential mode but without CoCo, and how to improve the interaction between the two projects along the way.
+
+Due to a large number of dependencies, the attack surface within the guest VM can grow quite large especially with  more features are enabled, which is a concern for users. While CoCo can be viewed as a downstream project from Kata, building a guest can require CoCo components, such as guest pool, attestation agent, which is now being used to verify the init data digest, and confidential data hub. These components are currently integrated in a seemingly ad-hoc way, and attendees used the available time during the session to explore how to make it more stable and clean.
+
+The idea that was brought up during the session was to turn the guest pool and attestation agent into binaries, which could only be executed if present in the deployment, without hardcoding how to access them. However these components are already binaries, currently the kata-agent serves as a process manager for these, which is not ideal. The community has been exploring to implement a systemd-based approach, which got discussed again during the session.
+
+The community agreed that the goal is to keep the Kata runtime lightweight and avoid pulling in unnecessary dependencies when used without CoCo. Next steps will include the implementation to use systemd, improve the flexibility on who and how can use attestation and monitor the init data implementation in CoCo to improve the integration between the two projects.
```

Comment:
> on who and how can use attestation

This reads a bit strange, maybe use "who can use attestation, and how"?

Reply:
Updated the text.